### PR TITLE
#164798241 Hound<>Github Integration

### PR DESCRIPTION
**Pull request by e-ian**
#### What does this PR do?
- Configures hound and eslint to check for style violations
#### Description of tasks to be completed.
- Add eslint configuration
- Add .hound.yml file to configure eslint
- Add eslint dependency to package.json
#### How should this be manually tested?
- Clone this repo :
   `git clone ah-frontend-prime`

- Git checkout to this branch :
   `git checkout ch-hound-git-hub-integration-164798241`

- Install all dependencies :
   `npm install `

- Run eslint:
   `npm run lint`
#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
[#164798241](https://www.pivotaltracker.com/story/show/164798241)
#### Screenshots (if appropriate)
image

### CODE REVIEW

**Review by anyatibrian — APPROVED** [review]
LGTM

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        },
+        "ecmaVersion": 2018,
+        "sourceType": "module"
+    },
+    "plugins": [
+        "react"
+    ],
+    "rules": {
+    }
+}
```
Comment (Genza999) [inline .eslintrc.json]: Hey @e-ian , Great work on your implementation. Kindly add final new lines on this file and .hound.yml . file.

Comment (Genza999) [inline .eslintrc.json]: Hey @e-ian Great work on your implementation. Though, kindly add a new line to this file and also .hound.yml file too

Comment (e-ian) [reply .eslintrc.json]: Noted @Genza999. Done!

**Review by patrickf949 — APPROVED** [review]
LGTM

File: .hound.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+eslint:
+  enabled: true
+  config_file: .eslintrc
+  ignore_file: .eslintignore
+
```
Comment (MuhanguziDavid) [inline .hound.yml]: Add a new line at the end of this file.

File: .eslintrc.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
+{
```
Comment (MuhanguziDavid) [inline .eslintrc.json:1]: Use indentation of two(2) spaces for this file and any other file.

File: .hound.yml (hunk 2 above)
Comment (e-ian) [reply .hound.yml]: I have added a new line @MuhanguziDavid.

File: .eslintrc.json (hunk 3 above)
Comment (e-ian) [reply .eslintrc.json:1]: All done @MuhanguziDavid

**Review by Genza999 — APPROVED** [review]
LGTM. Great work @e-ian

**Pull request merged**